azure.go

```diff
@@ -498,7 +498,7 @@ type batchRequest struct {
 	Method      string `json:"httpMethod"`
 }
 
-func resourceURLFrom(resource string, metricNames string, aggregations []string) string {
+func resourceURLFrom(resource string, metricNames string, aggregations []string, timespan int) string {
```

Comment:
You may want to change that to `delay_minutes` everywhere

Reply:
done